mlflow/sagemaker/__init__.py

```diff
@@ -600,6 +601,11 @@ def deploy_transform_job(
                    a flavor is automatically selected from the model's available flavors. If the
                    specified flavor is not present or not supported for deployment, an exception
                    will be thrown.
+    :param archive: If ``True``, resources associated with the specified batch transform job,
+                    such as its associated models and model artifacts, are preserved.
```

Comment:
@YQ-Wang Can we clarify that the resources are archived after the job terminates?